reddit/settings.py

```diff
@@ -148,3 +148,10 @@
 # http://django-crispy-forms.readthedocs.io/en/latest/install.html#template-packs
 CRISPY_TEMPLATE_PACK = "bootstrap5"
 CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap5"
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.gmail.com"
```

Comment:
Tiny remark, I don't know why I did not think about it earlier. Here we are assuming everybody will use gmail. 
Please move host, host port, and use tls into config too. 
Sorry for not catching it earlier.